jerry-core/ecma/base/ecma-alloc.c

```diff
@@ -94,6 +97,26 @@ DECLARE_ROUTINES_FOR (getter_setter_pointers)
 DECLARE_ROUTINES_FOR (external_pointer)
 
 /**
+ * Allocate memory for ecma-property pair
+ *
+ * @return pointer to allocated memory
```

Comment:
I'd keep doc comment as is. This function simply wraps `mem_heap_alloc_block`. Should its functionality change, e.g., may we prepare for OOM situations in the (not too near) future, then this function will change its behaviour as well. And then the suggested note would become invalid.